README.rst

```diff
@@ -13,3 +13,131 @@ __ https://discuss.openedx.org/c/working-groups/build-test-release/30
 More details about the working group are at the `BTR page in the wiki`__.
 
 __ https://openedx.atlassian.net/wiki/spaces/COMM/pages/1022099494/Build+-+Test+-+Release+Working+Group
+
+================================
+Role definitions and assignments
+================================
+
+*Current release: Lilac*
+
+The following roles, with corresponding expectations and assignments, are
+defined for the current release cycle.
+
+Group Chair
+===========
+
+*Current assignee: @arbrandes*
+
+The group chair is responsible for coordinating efforts that culminate in
+getting a new release out and maintaining the current one.  In particular, the
+Group Chair will:
+
+* Manage the current release cycle in general, including making sure that
+  proper deadlines are established and adhered to
+* Facilitate the assignment of roles and tasks
+* Coordinate the group's communication efforts, including running periodic
+  meetings such as the Biweekly Meetup, and summarizing recent happenings in
+  the Biweekly Update
+* Try and help solve problems as they come up, bringing them to the attention
+  of the right group members
+
+Release Manager
+===============
+
+*Current assignees: @nedbat and @arbrandes*
+
+The Release Manager is responsible for the actual cutting of a release, which,
+in essence, includes following the steps in the `Process to Create an Open edX Release
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/19662426/Process+to+Create+an+Open+edX+Release>`_.
+
+In addition, the Release Manager will:
+
+* Create public test or release candidate branches as required by the release process
+* Merge reviewed PRs into the release master branch (for instance,
+  ``koa.master``)
+* Notify the Release Testing Coordinator whenever additional testing is needed
+  in response to the above
+* Tag releases, major and minor, at the appropriate time
+
+Release Testing Coordinator
+===========================
+
+*Current assignee: @jfavellar90*
+
+The Release Testing Coordinator is responsible for making sure release
+candidates are tested _before_ a tag is made.  The coordinator does not
+necessarily have to run tests themselves, but they will:
+
+* Coordinate testing in general, in particular in the days leading to a timed
+  release, including the bimonthly minor ones.
+* Keep tabs on whatever CI is available, making sure it is running correctly
+* Advocate for better testing tools and environments, when necessary
+* Coordinate the reaction to test failures, such as bringing them to the
+  attention of the rest of the group by opening tickets or commenting in the forum
+
+Release Documentation Expert
```

Comment:
Talking about documentation, I think this group is going to be generating a kind of knowledge base, which is basically essential notes about every role operation (how to tag repositories, how to test releases, how to add release notes, etc). It should be included in this repo I think. Is this something worth discussing in this scope?

Reply:
+1

Reply:
@jfavellar90, I'm with you.  It should probably have it's own ADR.  For the most part, this will mean porting [the current wiki page](https://openedx.atlassian.net/wiki/spaces/COMM/pages/19662426/Process+to+Create+an+Open+edX+Release) here.  Created [an issue](https://github.com/openedx/build-test-release-wg/issues/33) for it.